scripts/prepare_binary_package.sh

```diff
@@ -105,7 +97,3 @@ filter-noassertion = true
 EOT
 
 cargo about generate about.hbs -o $target_path/index.html
-
-if [ "$2x" == "--with-qtx" ]; then
-    cp `dirname $0`/../internal/backends/qt/LICENSE.QT `dirname $0`/../internal/backends/qt/QtThirdPartySoftware_Listing.txt $target_path/
```

Comment:
We can as well delete these files form our repo, i suppose.
They are probably already outdated anyway.